label-application-rule-config-service-api/src/main/proto/org/hypertrace/label/application/rule/config/service/v1/label_application_rule_config_service.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package org.hypertrace.label.application.rule.config.service.v1;
+
+option java_multiple_files = true;
+
+import "org/hypertrace/label/application/rule/config/service/v1/label_application_rule.proto";
+
+service LabelApplicationRuleConfigService {
+
+  // create label application rule
+  rpc CreateLabelApplicationRule(CreateLabelApplicationRuleRequest) returns (CreateLabelApplicationRuleResponse) {}
+
+  // get label application rule by id
+  rpc GetLabelApplicationRule(GetLabelApplicationRuleRequest) returns (GetLabelApplicationRuleResponse) {}
+
+  // get all label application rules
+  rpc GetLabelApplicationRules(GetLabelApplicationRulesRequest) returns (GetLabelApplicationRulesResponse) {}
+
+  // update label application rule
+  rpc UpdateLabelApplicationRule(UpdateLabelApplicationRuleRequest) returns (UpdateLabelApplicationRuleResponse) {}
+
+  // delete label application rule by id
+  rpc DeleteLabelApplicationRule(DeleteLabelApplicationRuleRequest) returns (DeleteLabelApplicationRuleResponse) {}
+
+}
+
+message CreateLabelApplicationRuleRequest {
+  CreateLabelApplicationRule label_application_rule = 1;
```

Comment:
We already have a create-specific `CreateLabelApplicationRuleRequest`, we don't need `CreateLabelApplicationRule`, we can just unwrap it and have a `LabelApplicationRuleData`

Reply:
Sure